pyconcz/templates/__nav-link.html

```diff
@@ -7,22 +7,23 @@
 {% endif %}
 
 {% if link %}
-    {% if request.path != '/2019/' or  link != '/2019/' or extra %} {# hide link to homepage on homepage #}
-        <a
-            class="
-                nav-link
-                {{ class }}
-                {% if request.path == link and not extra %}active{% endif %}
-            "
-            {% if request.path == link and not extra %}
+    <a
+        class="
+            nav-link
+            {% if request.path == '/2019/' and  link == '/2019/' or extra %} {# hide link to homepage on homepage #}
```

Comment:
It was `or extra` and that has to change to `and not extra` to keep the same functionality.

Reply:
`extra` is there for the case that the link goes to homepage BUT is with anchor (`#sponsors` in this case) so it's not really a link we want to treat as a homepage link.

Other option is to solve this more straightforward just by adding another parametr `is_homepage` (like `extra` is) which would determine what is and what is not a homepage link, the condition would be much clearer.